### PR TITLE
Added L0 tests.

### PR DESCRIPTION
**Pull request by sundargs2000**
Wrote L0 tests as a part of this [task](https://github.com/github/GitHub-EcoSystem/issues/1200#issue-814239231).

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -1,81 +1,21 @@
-on:
+name: "build-test"
```
Comment (ds-ms) [inline .github/workflows/test.yml:1]: Have a separate workflow for L0s. The existing one is for L2s

Comment (ds-ms) [reply .github/workflows/test.yml:1]: If the L2 e2e is not fully setup, then it's okay.

Comment (sundargs2000) [reply .github/workflows/test.yml:1]: L2 tests haven't been setup.

File: __tests__/run.test.ts [hunk 2]
```diff
@@ -1,7 +1,104 @@
...
+        expect(toolCache.downloadTool).toBeCalled();
+    });
+
+    test('downloadKubectl() - return path to existing cache of kubectl', async () => {
+        jest.spyOn(toolCache, 'find').mockReturnValue('pathToCachedTool');
+        jest.spyOn(os, 'type').mockReturnValue('Windows_NT');
+        jest.spyOn(fs, 'chmodSync').mockImplementation(() => {});
+        
+        expect(await run.downloadKubectl('v1.15.0')).toBe('pathToCachedTool/kubectl.exe');
+        expect(toolCache.find).toBeCalledWith('kubectl', 'v1.15.0');
```
Comment (shigupt202) [inline __tests__/run.test.ts:100]: You can add an `expect` here to check that toolCache.downloadTool doesn't get called using `.not.toBeCalled`.

**Review by shigupt202 — APPROVED** [review]

**Pull request closed without merging**